src/components/interactive-builder/edit-question.modal.tsx

```diff
@@ -91,6 +85,12 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
   const { t } = useTranslation();
   const { fieldTypes, questionTypes } = useConfig<ConfigObject>();
 
+  const datePickerTypeOptions: DatePickerTypeOptions = {
+    datetime: [{ value: 'both', label: t('calendarAndTimer', 'Calendar and timer'), defaultChecked: true }],
+    date: [{ value: 'calendar', label: t('calendarOnly', 'Calendar only'), defaultChecked: false }],
+    time: [{ value: 'timer', label: t('timerOnly', 'Timer only'), defaultChecked: false }],
+  };
```

Comment:
Can we get rid of the code duplication here?

Reply:
Trying to refactor this is a bit tough because of the translation strings. Any suggestions?